src/components/create_cluster/index.js

```diff
@@ -586,11 +586,30 @@ function mapStateToProps(state) {
   var maxAvailabilityZones = state.app.info.general.availability_zones.max;
   var selectedOrganization = state.app.selectedOrganization;
   var provider = state.app.info.general.provider;
-  var defaultInstanceType = 'm3.large'; // TODO
-  var defaultVMSize = 'Standard_A2_v2'; // TODO
+
+  var defaultInstanceType;
+  if (
+    state.app.info.workers.instance_type &&
+    state.app.info.workers.instance_type.default
+  ) {
+    defaultInstanceType = state.app.info.workers.instance_type.default;
+  } else {
+    defaultInstanceType = 'm3.large';
+  }
+
+  var defaultVMSize;
+  if (
+    state.app.info.workers.vm_size &&
+    state.app.info.workers.vm_size.default
+  ) {
+    defaultVMSize = state.app.info.workers.vm_size.default;
+  } else {
+    defaultVMSize = 'Standard_D2s_v3';
+  }
+
   var defaultCPUCores = 1; // TODO
   var defaultMemorySize = 1; // TODO
-  var defaultDiskSize = 1; // TODO
+  var defaultDiskSize = 20; // TODO
```

Comment:
why `20`?

Reply:
I think that was the default it was originally.